documentation/guides/draft/rust/get-started/xx-discovery-using-cloud-node/README.md

```diff
@@ -0,0 +1,137 @@
+# Discovery using cloud node
+
+## Introduction
+
+In the previous guide we learned how to [create a cloud node](../xx-cloud-node) and connect to it using the TCP transport.
+
+Cloud nodes can be used as proxies to connect devices not exposed by public hostnames or IPs. In this case the cloud node would redirect the traffic from one node to another using the transport connections established by devices.
+
+## Discovery using forwarding
+
+In this guide we're going to use the "forwarding_service" - a Hub Node service which allows a device to connect to another device using a temporary route alias.
+
+In order to do that, a device we call "responder" is going to create an alias first, which can be used by another device we call "initiator" to send messages to responder.
+
+After discovery step both devices would have a route to each other, which they can use to exchange messages.
+
+<img src="./Discovery.png" width="70%">
```

Comment:
First place we introduce the terminology `alias`. I think this is a nice succinct word better than "forwarding address"